File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/networking/identify/IdentifyResponse.java
package com.yelloco.fingodriver.models.networking.identify;

import com.google.gson.annotations.SerializedName;

public class IdentifyResponse
{
    @SerializedName("kan-dev:" + "fp_member_id")
    private String memberId;

    @SerializedName("kan-dev:" + "fp_veinid_id")
    private String veinId;

    public IdentifyResponse() {
    }

    public IdentifyResponse(String memberId, String veinId) {
        this.memberId = memberId;
        this.veinId = veinId;
    }

    public String getMemberId() {
        return memberId;
    }

    public void setMemberId(String memberId) {
        this.memberId = memberId;
    }

    public String getVeinId() {
        return veinId;
    }

    public void setVeinId(String veinId) {
        this.veinId = veinId;
    }
}

File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/networking/identify/IdentifyResponse.kt
package com.yelloco.fingodriver.models.networking.identify

import com.google.gson.annotations.SerializedName

class IdentifyResponse
{
    @SerializedName("kan-dev:" + "fp_member_id")
    var memberId: String? = null

    @SerializedName("kan-dev:" + "fp_veinid_id")
    var veinId: String? = null

    constructor() {}
    constructor(memberId: String?, veinId: String?) {
        this.memberId = memberId
        this.veinId = veinId
    }

    override fun toString(): String {
        return "IdentifyResponse(memberId=$memberId, veinId=$veinId)"
    }


}